transformation-builder/src/test/kotlin/com/cloudinary/transformation/TransformationTest.kt

```diff
@@ -349,4 +350,14 @@ class TransformationTest {
                 })
         })
     }
+
+    @Test
+    fun testQualityAlias() {
```

Comment:
Maybe `testAliases`?
The test also includes `format`, which is not represented in the test name.

Reply:
Fixed